library/syscalls.c

```diff
@@ -33,6 +33,21 @@ int object_type(int fd)
 	return syscall(SYSCALL_OBJECT_TYPE, fd, 0, 0, 0, 0);
 }
 
+int process_object_max()
+{
+	return syscall(SYSCALL_PROCESS_OBJECT_MAX, 0, 0, 0, 0, 0);
+}
+
+char *object_set_intent(int fd, char *intent)
```

Comment:
Return integer reflecting success/failure.

Reply:
Updated get_intent and set_intent to return integer codes instead of char *